### PR TITLE
Added Gujarati locale for moment

### PR DESCRIPTION
**Pull request by Kaushik1987**
Hi there,

I have added required files for Gujarati local please let me know if I missed something.  This is my first contribution for any git repository

### CODE REVIEW

**Comment by icambron** [comment]
This looks fine, but Gujarati is`gu`, not `gj`

**Comment by Kaushik1987** [comment]
Will update file name & culture variable if required to "gu".

**Comment by Kaushik1987** [comment]
@marwahaha can you help me now ? I m  not sure what went wrong

File: src/locale/gu-IN.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+//! moment.js locale configuration
+//! locale : gujarati (gj)
```
Comment (icambron) [inline src/locale/gu-IN.js]: this should say `gu` too

**Comment by icambron** [comment]
I suspect the test failure was transient; I'm rerunning it. I also left one small comment on the code. When that's all cleared up, I'll flag this for inclusion in the next release.

**Comment by icambron** [comment]
Cool, thanks!

**Pull request merged**